Lib/test/test_faulthandler.py

```diff
@@ -168,6 +171,42 @@ def test_sigsegv(self):
             3,
             'Segmentation fault')
 
+    @skip_segfault_on_android
+    def test_gc(self):
```

Comment:
Please, are two more tests:

* gc running on a different thread
* gc not running

Reply:
The "gc not running" case is covered by all other tests in test_faulthandler. The file contains 45 tests.

Reply:
> The "gc not running" case is covered by all other tests in test_faulthandler.

The check_error() helper function in tests uses a strict regular expression matching the exact output. It fails if `Garbage-collecting` is found of the output.